CONTRIBUTING.md

```diff
@@ -34,6 +34,28 @@ _Before_ submitting a pull request, please make sure the following is done…
 
     Note: Replace `<your_username>` with your GitHub username
 
+1.  Run Jest with [Docker](https://docs.docker.com/engine/docker-overview/) (Optional) or continue reading for manual instructions.
+
+    1. If you haven't already, [Install Docker](https://docs.docker.com/install/).
+
+    1. Navigate to the `/jest` directory. You should be able to see the `Dockerfile` file when you run `ls`.
```

Comment:
Not sure if this line is needed. Is this the directory with cloned repo or something else? In the previous step ("Fork the repo and create your branch..."), a user was suggested to run `cd jest`.

Reply:
Agree.